### PR TITLE
Initial Implementation of dateDisabled attribute.

### PR DESCRIPTION
**Pull request by lmudhaliyar**
Initia Implementation of dateDisabled attribute.

### CODE REVIEW

**Comment by Dinistro** [comment]
@lmudhaliyar Please fix the linting issues. You can see more information in the travis logs

File: components/datepicker/datepicker-inner.component.ts [hunk 1]
```diff
@@ -310,7 +310,16 @@ export class DatePickerInnerComponent implements OnInit, OnChanges {
   }
 
   private isDisabled(date:Date):boolean {
-    // todo: implement dateDisabled attribute
+    if(this.dateDisabled !==undefined && this.dateDisabled!==null){
+		for(var i=0;i<this.dateDisabled.length;i++){
+			var disabledDate = this.dateDisabled[i].getDate() + "/" +(this.dateDisabled[i].getMonth()+1) + "/" +(this.dateDisabled[i].getFullYear());
```
Comment (mrdav30) [inline components/datepicker/datepicker-inner.component.ts:315]: If you change this.dateDisabled[i] to this.dateDisabled[i].date for each of your function calls this works perfectly.

**Comment by mrdav30** [comment]
@lmudhaliyar using your code works perfectly to disable dates if you change your iteration functions calls from this.dateDisabled[i] to this.dateDisabled[i].date

**Comment by valorkin** [comment]
closing for inactivity but will take in count when merging new datepicker version

**Pull request closed without merging**